truss-chains/truss_chains/stub.py

```diff
@@ -81,17 +82,20 @@ def _client_sync(self) -> httpx.Client:
         assert self._cached_sync_client is not None
         return self._cached_sync_client[0]
 
-    async def _client_async(self) -> httpx.AsyncClient:
+    async def _client_async(self) -> aiohttp.ClientSession:
         # Check `_client_cycle_needed` before and after locking to avoid
         # needing a lock each time the client is accessed.
         if self._client_cycle_needed(self._cached_async_client):
             async with self._async_lock:
                 if self._client_cycle_needed(self._cached_async_client):
+                    connector = aiohttp.TCPConnector(
+                        limit=self._client_limits.max_connections,
```

Comment:
> nit: is max_connections still 1000? wondering if we should have a lower number since we saw decreased performance at 1000?

It is -- I think the tough thing with this is that the right number is highly dependent on workload/hardware run. I am strongly leaning towards keeping this high & leaving it up to users to limit the # of outgoing connections.

Reply:
yeah that makes sense. 👍 Probably something we should document somewhere, either in docs or truss example (or both)